es6-async-await/exceptions-best.js

```diff
@@ -0,0 +1,69 @@
+import read from './read.js';
+
+// Keep track of app load time so each log message can be timed.
+// Log messages should all be approximately 1 second apart.
+const startTime = Date.now();
+const elapsed = () => `${Math.round((Date.now() - startTime) / 1000)}s -`;
+
+async function throwOnce() {
+  // Note: In the `catch` we are logging just `error.message` for illustration
+  // purposes. In actual code you will want to log the entire error so that
+  // you get the stack trace.
+
+  const response = await read('foo', true);
+  if (!response) {
+    throw response;
```

Comment:
You usually wouldn't just `throw response`, especially once you've determined that `response` is falsy. Instead you would `throw new Error('some message');`